src/plugins/walletConnectToDapps/typeGuards.ts

```diff
@@ -55,5 +55,11 @@ export const isTransactionParams = (
   Boolean(
     (transaction.gasLimit !== undefined && transaction.gasPrice !== undefined) ||
       transaction.gas !== undefined ||
-      (transaction.maxFeePerGas !== undefined && transaction.maxPriorityFeePerGas !== undefined),
+      (transaction.maxFeePerGas !== undefined && transaction.maxPriorityFeePerGas !== undefined) ||
+      // All gas params undefined is also a valid payload
```

Comment:
Does what it says on the comment - the modal firing is a nice fix, but not so nice if it's in an infinite loading state - all gas params missing *is* valid